composer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "wordpress/twentytwenty",
+  "description": "Twenty Twenty is designed with flexibility at its core.",
+  "type": "package",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "WordPress",
+    "Themes"
+  ],
+  "authors": [
+    {
+      "name": "the WordPress team",
+      "homepage": "https://github.com/WordPress/twentytwenty"
```

Comment:
Should this link to the `contributors` page on GH as it is supposed to be the _Homepage of the authors_ ?

You can add a top-level `homepage` item for the project homepage.

Refs:
* https://getcomposer.org/doc/04-schema.md#homepage
* https://getcomposer.org/doc/04-schema.md#authors